integration/networking/bridge_test.go

```diff
@@ -675,3 +675,31 @@ func TestDisableIPv6Addrs(t *testing.T) {
 		})
 	}
 }
+
+// Test that it's possible to set a sysctl on an interface in the container.
+// Regression test for https://github.com/moby/moby/issues/47619
+func TestSetInterfaceSysctl(t *testing.T) {
+	skip.If(t, testEnv.DaemonInfo.OSType == "windows", "no sysctl on Windows")
```

Comment:
For a follow-up; all tests in this file (`bridge_test.go`) are Linux-only; if this file is specific for the `bridge` driver (which is not supported on Windows), perhaps we should consider renaming the file to `bridge_linux_test.go`, or adding a `//go:build linux` build-tag to it.

Reply:
It's got `TestBridgeICCWindows` - which uses "nat" instead of "bridge", I think I put it there because it covers some of the same ground as `TestBridgeICC` ... but yes, we could split the file in to windows/not-windows.